docs/machine-requirements.md

```diff
@@ -6,7 +6,7 @@ Whether you want to contribute to Aspire, or just use the latest build of Aspire
 
 On Windows, Visual Studio contains special tooling support for .NET Aspire that you will want to have.
 
-[Visual Studio 2022 version 17.10 Preview](https://visualstudio.microsoft.com/vs/preview/)
+[Visual Studio 2022 version 17.11 Preview](https://visualstudio.microsoft.com/vs/preview/)
```

Comment:
mmm this is not actually a requirement, as you should be able to use Aspire with 17.10. Should we make a note saying that 17.10 is the minimum requirement (and add the right link to it of course), and then say that 17.11 is preferred for newer features?

Reply:
This is to build Aspire, not use it. Are we confident we've got good coverage of using non-preview bits to build the repo? If so, seems fine to change to your suggestion.

Reply:
I believe so, at least I've been working with 17.10 GA and building the repo and it has worked fine for me. In any case, I did originally believed that this was for consumers of Aspire (as opposed to contributors) so I'm fine with what you prefer.